src/lib/pay-request/services/connector/types.ts

```diff
@@ -48,7 +48,7 @@ export interface GatewayAccount {
   send_reference_to_gateway: boolean;
   email_collection_mode?: EmailCollectionMode,
   email_notifications: EmailNotifications,
-  notify_settings: NotifySettings
+  notifySettings: NotifySettings
```

Comment:
Why has this been changed? Going by the lines above and below it seems snake case has been settled on.

Reply:
Ah because this was actually just wrong - and connector returns all the fields as snake case except `notifySettings` and requires3ds